test/Fields/reduction_cuda.jl

```diff
@@ -93,5 +95,24 @@ end
     # test minimum
     @test local_min[1] ≈ FT(0)
     @test local_min[1] ≈ local_min_cpu
+    # testing mean
+    meanz = Statistics.mean(Z)
+    meanz_cpu = Statistics.mean(Z_cpu)
+    @test meanz[] ≈ meanz_cpu[]
```

Comment:
doesn't this return a scalar? why do you need `[]`?

Reply:
The cuda version returns a Vector of size `Nf`.